test/task-filtering.test.js

```diff
@@ -0,0 +1,94 @@
+import { filterTasksByType, filterTasksByStatus, filterTasksByCreationTime } from '../src/utils/task-filtering';
+import mockTasks from '../src/mock/mock-tasks.json';
+
+describe('Task Filtering Functions', () => {
```

Comment:
I would recommend adding comments for each test case here explaining what it is testing for, what type of test it us (error handling, functionality test, etc), and what the expected outcome is

Reply:
You kind of already have the information in the code itself but it would be better to specifically spell it out

Reply:
As Shayan said, please add at least one comment per test case discussing what kind of test it is.